web/template/layout/fixtures-page.gohtml

```diff
@@ -0,0 +1,43 @@
+{{ template "page" . }}
+
+{{ define "main" }}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <ul class="govuk-list">
+        <li><a class="govuk-link govuk-link--no-visited-state" href="{{ link .App .App.Paths.Fixtures.Format }}">Donor</a></li>
+        <li><a class="govuk-link govuk-link--no-visited-state" href="{{ link .App .App.Paths.CertificateProviderFixtures.Format }}">Certificate provider</a></li>
+        <li><a class="govuk-link govuk-link--no-visited-state" href="{{ link .App .App.Paths.AttorneyFixtures.Format }}">Attorney</a></li>
+        <li><a class="govuk-link govuk-link--no-visited-state" href="{{ link .App .App.Paths.DashboardFixtures.Format }}">Dashboard</a></li>
+      </ul>
+
+      <form novalidate method="post">
+        <h1 class="govuk-heading-xl">{{ template "pageTitle" . }}</h1>
+      
+        {{ template "fixtures-main" . }}
+        <button type="submit" class="govuk-button" data-module="govuk-button">Start</button>
+
+        <div class="govuk-inset-text">
+          <p id="the-link"></p>
+        </div>
+
+        {{ template "csrf-field" . }}
+      </form>
+    </div>
+  </div>
+
+  <script>
+    const form = document.querySelector('form');
```

Comment:
You may want to scope this to the fixtures form as the cookie consent form gets selected if its not been accepted already (everything works fine once cookies are accepted)